tests/bluetooth/bsim_bt/edtt_ble_test_app/common/commands.h

```diff
@@ -235,7 +235,9 @@ enum commands_t {
 	CMD_GAP_READ_IRK_REQ,
 	CMD_GAP_READ_IRK_RSP,
 	CMD_GAP_ROLE_REQ,
-	CMD_GAP_ROLE_RSP
```

Comment:
This doesn't really seems to belong in this PR

Reply:
Well, no but then again, it is what is needed to have edtt support for set min used chans

Reply:
Also why is the values set to 255, that doesn't seems to follow any pattern in this enum?

Reply:
yeah, you're right - I took it out again. Once I get around to cleaning up the EDTT end of this, I'll add them back and sort out the numbering. The 255 is the next command avail, I think somehow this enum is not entirely up-to-date